src/EditorFeatures/CSharpTest/UseConditionalExpression/UseConditionalExpressionForReturnTests.cs

```diff
@@ -600,6 +603,34 @@ int M()
             yield return 1;
         }
     }
+}");
+        }
+
+        [WorkItem(27960, "https://github.com/dotnet/roslyn/issues/27960")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseConditionalExpression)]
+        public async Task TestWithNoElseBlockButFollowingYieldReturn()
+        {
+            await TestInRegularAndScriptAsync(
```

Comment:
❗️ 🐛 This should not be offered. The result is not correct.

Reply:
good catch. I think it shouldn't be offered.

Reply:
Oh, you're right.  good catch!

Reply:
Fixed.